packages/@uppy/dashboard/src/components/FileCard/index.js

```diff
@@ -95,13 +87,14 @@ class FileCard extends Component {
     const showEditButton = this.props.canEditFile(file)
 
     return (
-      <div
+      <form
```

Comment:
The reason we use less semantic stuff in Uppy and don‘t use `<form>` is because some people embed Uppy in a form, which might lead to conflicts. As far as I know, you can’t nest forms.

Reply:
This is required in get anything from `required` input, the browser UI only shows when the form is submitted. I can switch to `form-target` and add a Uppy `<form>` element in the body so this logic works even if we're already inside a `<form>`.

Reply:
Yes please, I guess we should try this way?

Reply:
Done in https://github.com/transloadit/uppy/pull/2896/commits/007d27656542a9f0849bd5592954bc6795bda746.